src/lib/modernExtend.ts

```diff
@@ -410,15 +417,20 @@ export function enumLookup(args: EnumLookupArgs): ModernExtend {
     return {exposes: [expose], fromZigbee, toZigbee, configure, isModernExtend: true};
 }
 
+// reverseToZigbee provides a way to distinguish between fromZigbee (default) and toZigbee value conversions if they are asymmetrical
+export type ScaleFunction = (value: number, reverseToZigbee?: boolean) => number;
```

Comment:
```suggestion
export type ScaleFunction = (value: number, type: 'from' | 'to') => number;
```

I would make this a bit more explicit

Reply:
Done